### PR TITLE
[FIX] various user facing an non-user-facing typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell`  
Note: I hereby relinquish any ownership of this PR an the fixes within. Please cherrypick this commit. I need no attributions associated with odoo.

### CODE REVIEW

File: odoo/modules/migration.py [hunk 1]
```diff
@@ -30,8 +30,8 @@ class MigrationManager(object):
     """
         This class manage the migration of modules
         Migrations files must be python files containing a `migrate(cr, installed_version)`
-        function. Theses files must respect a directory tree structure: A 'migrations' folder
-        which containt a folder by version. Version can be 'module' version or 'server.module'
+        function. These files must respect a directory tree structure: A 'migrations' folder
+        which contain a folder by version. Version can be 'module' version or 'server.module'
```
Comment (odony) [inline odoo/modules/migration.py:34]:
```suggestion
        which contains a folder by version. Version can be 'module' version or 'server.module'
```

File: doc/cla/corporate/maao.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
-Management and Accouting online llc agrees to the terms of the Odoo Corporate Contributor License Agreement v1.0.
+Management and Accounting online llc agrees to the terms of the Odoo Corporate Contributor License Agreement v1.0.
 
 I declare that I am authorized and able to make this agreement and sign this declaration.
 
 Signed,
 
-Management and Accouting online llc info@maao.com.ua https://github.com/management-and-acounting-on-line
+Management and Accounting online llc info@maao.com.ua https://github.com/management-and-acounting-on-line
```
Comment (odony) [inline doc/cla/corporate/maao.md:9]: I wouldn't modify the CLA of a contributor, regardless of any typos

**Comment by odony** [comment]
> Found via `codespell`

Thanks so much, this is really appreciated!

> Note: I hereby relinquish any ownership of this PR an the fixes within. Please cherrypick this commit. I need no attributions associated with odoo.

Unfortunately I don't think we can do that legally. There are jurisdictions where a copyright owners cannot discard their own copyright, and we can't afford any legal liability because of this.

Would it be a real problem for you to sign our CLA? 

You can do so with your anonymous github email, all need is to add a small CLA signature file in the doc/ folder as part of your PR, and you're done ([full explanation](https://github.com/odoo/odoo/blob/14.0/doc/cla/sign-cla.md))

File: odoo/service/server.py [hunk 3]
```diff
@@ -1034,7 +1034,7 @@ def run(self):
                          len(odoo.modules.registry.Registry.registries))
             self.stop()
         except Exception:
-            _logger.exception("Worker (%s) Exception occured, exiting..." % self.pid)
+            _logger.exception("Worker (%s) Exception occurred, exiting..." % self.pid)
```
Comment (mart-e) [inline odoo/service/server.py:1037]:
```suggestion
            _logger.exception("Worker (%s) Exception occurred, exiting...", self.pid)
```

**Comment by luzpaz** [comment]
@odony I've signed the CLA (50d2bf0)

File: doc/cla/corporate/maao.md (hunk 2 above)
Comment (luzpaz) [reply doc/cla/corporate/maao.md]: Reverted in 39aa2d2

File: odoo/service/server.py (hunk 3 above)
Comment (mart-e) [reply odoo/service/server.py:1037]: @luzpaz not sure if intended but it looks like this code is back in your latest code

Comment (luzpaz) [reply odoo/service/server.py]: sorry, i'm confused as to how. neverthless, fixed and

**Comment by mart-e** [comment]
Thanks for the update, it looks good.
I squashed the commits
@robodoo r+ rebase-ff

**Pull request closed without merging**